en/physics-2d/physics-2d-system.md

```diff
@@ -0,0 +1,158 @@
+# Physics Manager
+
+The physical system hides most of the implementation details of the physical modules (Box2D and Builtin modules) (e.g. creating rigid bodies, synchronizing rigid body information to nodes, etc.).
+
+You can use the physical system to access some of the common functions of the physics module, such as click testing, ray testing, and setting up test messages.
+
+## Physics system related settings
+
+### Enable Physics manager
+
+The physics manager is disabled by default. If you need to use the physics system related functions, the first thing you need to do is enable the physics manager, otherwise all the physics object you setup in the editor will not produce any effect.
```

Comment:
The physics manager is disabled by default. If you need to use the physics system related functions, the first thing you need to do is enable the physics manager, otherwise all the physics object you setup in the editor will not produce any effect.
-->
The __Physics Manager__ is disabled by default. To use the physics system-related functions, first, enable the __Physics Manager__, otherwise, all the physics objects setup in the editor will not produce any effect.